app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -279,7 +290,19 @@ protected void onCreate(Bundle savedInstanceState) {
                 .setAudioFormat(AUDIO_FORMAT_OUT_MONO)
                 .setBufferSizeInBytes(outputBufferSize)
                 .build();
+        // Set volume from SharedPreferences
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
+        float initVolume = preferences.getFloat(PREF_CURRENT_VOLUME, maxVolume);
+        Log.i(TAG, "setting volume to: " + initVolume);
+        mAudioTrack.setVolume(initVolume);
+        ASSISTANT_AUDIO_RESPONSE_CONFIG = AudioOutConfig.newBuilder()
+                .setEncoding(ENCODING_OUTPUT)
+                .setSampleRateHertz(SAMPLE_RATE)
+                .setVolumePercentage(Math.round(initVolume))
```

Comment:
We need to use `initVolume * 100.0f / maxVolume` to make sure it is scaled correctly b/w the internal sys volume and Assistant volume